src/dotnet/APIView/APIViewWeb/Controllers/PullRequestController.cs

```diff
@@ -102,7 +103,8 @@ public class PullRequestController : Controller
             string baselineCodeFileName = null,
             bool commentOnPR = true,
             string language = null,
-            string project = "public")
+            string project = "public",
+            string sourceBranch = null)
```

Comment:
Won't this source branch always be PR branch in the format `refs/pull/<PR number>/merge` in case of PR API revision request? We have a PR number param so what's the use of this additional param that will send pretty much same data.